<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoolListWidgetItem</class>
 <widget class="QWidget" name="PoolListWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_poolName">
     <item>
      <widget class="QLabel" name="label_poolName">
       <property name="text">
        <string>Pool</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_protocol_fee">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_protocol">
       <property name="text">
        <string>Protocol:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_protocol">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_poolFee">
       <property name="text">
        <string>Pool Fee (%):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_poolFee">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_spotPrice">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_spotPrice">
       <property name="text">
        <string>Spot price against input token: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_spotPrice"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_poolTable">
     <item>
      <widget class="QTableWidget" name="tableWidget_poolInformation">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>Current Values</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Delta</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
